edgehog-device-runtime-store/migrations/2024-11-29-163409_initial/up.sql

```diff
@@ -0,0 +1,166 @@
+-- This file is part of Edgehog.
+--
+-- Copyright 2025 SECO Mind Srl
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--    http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+-- SPDX-License-Identifier: Apache-2.0
+
+CREATE TABLE IF NOT EXISTS images (
+    id BLOB PRIMARY KEY NOT NULL,
+    local_id TEXT,
+    status INTEGER NOT NULL,
+    reference TEXT NOT NULL,
+    registry_auth TEXT
+);
+
+CREATE TABLE IF NOT EXISTS networks (
+    id BLOB PRIMARY KEY NOT NULL,
+    local_id TEXT,
+    status INTEGER NOT NULL,
+    driver TEXT NOT NULL,
+    internal BOOLEAN NOT NULL,
+    enable_ipv6 BOOLEAN NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS network_driver_opts (
+    network_id BLOB NOT NULL,
+    name TEXT NOT NULL,
+    value TEXT,
+    PRIMARY KEY (network_id, name),
+    FOREIGN KEY (network_id) REFERENCES networks (
+        id
+    ) ON DELETE CASCADE ON UPDATE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS volumes (
+    id BLOB PRIMARY KEY NOT NULL,
+    status INTEGER NOT NULL,
+    driver TEXT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS volume_driver_opts (
+    volume_id BLOB NOT NULL,
+    name TEXT NOT NULL,
+    value TEXT,
+    PRIMARY KEY (volume_id, name),
+    FOREIGN KEY (volume_id) REFERENCES volume (
```

Comment:
The table name should be `volumes`
```suggestion
    FOREIGN KEY (volume_id) REFERENCES volumes (
```

Reply:
Thanks good catch. Unfortunately, SQLite doesn't error when there's a missing table constraint. So there is no easy to check this type of error automatically